app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Add new Recipe</h1>
+
+ <%= form_with model: @recipe_new, url: recipe_recipe_foods_path do |f| %>
+  <div class="form_control">
+      <%= f.label :quantity %>
+      <%= f.number_field :quantity %>
+    </div>
+
+    <div class="form_control">
+    <%= f.select(:food_id, @foods.map{|u| [u.name, u.id]}) %>
+    </div>
+
+    <div class="form_control">
+    <%= f.hidden_field :recipe_id, value: @recipe_new.recipe_id %>
+    </div>
+    <%= f.submit "Submit", class:"btn btn-success" %>
+<% end%>
```

Comment:
- [ Required ] To test your `App` I created two users, added a public `recipe`, and logged out of the application, after which I logged in with a different user. I then tried to add an ingredient to a `recipe` owned by another user and it succeeded, I believe this should not be the desired outcome. Please consider fixing this.

## Project requirements
![Screenshot from 2023-02-18 15-13-42](https://user-images.githubusercontent.com/16392046/219865204-0ab52f4a-52eb-4045-83f6-c252516551aa.png)

Reply:
Thank you very much. We will add that shortly